integration-tests/bats/reflog.bats

```diff
@@ -1,56 +1,231 @@
 #!/usr/bin/env bats
 load $BATS_TEST_DIRNAME/helper/common.bash
 
+setup() {
+    setup_common
```

Comment:
This was originally put into the tests (as opposed to in a `setup` function) so that the remote/local engine tests could run (we need to set the env vars before the `setup_common` code runs, otherwise the sql-server is already started and it's too late. That would probably have been a good comment! 😉 ). I noticed that those remote engine tests were turned off for the reflog bats test. I think it's generally better to keep them on if we at all can so that we have more testing coverage, but if you think they really shouldn't run for reflog let me know and I'm happy to chat more about it.